patches/minecraft/net/minecraft/data/recipes/RecipeCategory.java.patch

```diff
@@ -0,0 +1,21 @@
+--- a/net/minecraft/data/recipes/RecipeCategory.java
++++ b/net/minecraft/data/recipes/RecipeCategory.java
+@@ -1,6 +_,6 @@
+ package net.minecraft.data.recipes;
+ 
+-public enum RecipeCategory {
++public enum RecipeCategory implements net.minecraftforge.common.IExtensibleEnum {
+    BUILDING_BLOCKS("building_blocks"),
+    DECORATIONS("decorations"),
+    REDSTONE("redstone"),
+@@ -19,5 +_,10 @@
+ 
+    public String m_247710_() {
+       return this.f_244582_;
++   }
++
++   public static RecipeCategory create(String name, RecipeCategory fieldName)
```

Comment:
This is not a valid constructor. `IExtensibleEnum` for runtime injection takes in a string representing the name of the constant first, then the parameters of the enum constructor it is representing.